test/ExecCommand.js

```diff
@@ -125,6 +125,7 @@ describe("ExecCommand", () => {
             env: expect.objectContaining({
               LERNA_PACKAGE_NAME: "package-2",
             }),
+            reject: true,
```

Comment:
Implementing defaults actually caught this issue where this *should* have been passed (`bail` defaults to `true`) but was never actually defaulting to true due to the way we're testing it by bypassing yargs (we pass flags directly to the constructor).